src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -79,4 +79,17 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
 
         return $filter & Caster::EXCLUDE_VERBOSE ? $z : $z + $a;
     }
+
+    /**
+     * Formats seconds.
+     *
+     * @param string $s  seconds
+     * @param string $us microseconds
+     *
+     * @return string formatted seconds
+     */
+    private static function formatSec($s, $us)
```

Comment:
sorry, last comment: formatSeconds (then, I don't see much value in the docblock, so if you want you can remove it :) )